tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -105,38 +105,45 @@ internal static class Debugger
             public const string RedactedTypes = "DD_DYNAMIC_INSTRUMENTATION_REDACTED_TYPES";
 
             /// <summary>
-            /// Configuration key for enabling or disabling Exception Debugging.
+            /// [Deprecated] The old configuration key for enabling or disabling Exception Replay.
```

Comment:
You could/should potentially add the `[Obsolete]` attribute?

Given this is still in beta, do we need to keep the old configuration keys at all? I guess so, given this has _just_ changed

Reply:
We decided internally to keep both names. We already have docs out there that mention the old way of enabling it, so we decided to keep it for now